src/playwright-runner.ts

```diff
@@ -299,20 +305,20 @@ async function getCfg(runCfgPath, suiteName) {
   if (!runCfg.sauce) {
     runCfg.sauce = {};
   }
-  runCfg.sauce.region = runCfg.sauce.region || 'us-west-1';
+  // runCfg.sauce.region = runCfg.sauce.region || 'us-west-1';
```

Comment:
:o?

Reply:
Oops, forgot to clean that up. I just moved this logic to https://github.com/saucelabs/sauce-playwright-runner/pull/207/files#diff-2aa322c060bf9092e8747f9037d13117131aea4557f8397985c67a4241504722R263

Reply:
I just uncommented this line as well in addition to another nullish check linked above.